### PR TITLE
Array methods slides

### PR DESCRIPTION
**Pull request by nachosoloaga**

### CODE REVIEW

File: src/slides/class3/slides/ArrayMethods.jsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+      </Text>
+    </Slide>
+
+    <Slide backgroundColor="#0d0d0d">
+      <Heading margin="0px" fontSize="50px">
+        Array.map
+      </Heading>
+
+      <Text fontSize={30}>
+        El método <strong>map()</strong> crea un nuevo array con los resultados
```
Comment (JulianPasquale) [inline src/slides/class3/slides/ArrayMethods.jsx:57]: Can we add a link to the doc of each method? So if people is curious, they can go to the docs and read more about any method

File: src/slides/class3/slides/ArrayMethods.jsx [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    </Slide>
+
+    <Slide backgroundColor="#0d0d0d" style={{ display: "flex" }}>
+      <Heading margin="0px" fontSize="50px">
+        Array.find()
+      </Heading>
+
+      <Text fontSize={30}>
+        El método <strong>find()</strong> devuelve el valor del primer elemento
+        del array que cumple la función de prueba proporcionada.
```
Comment (JulianPasquale) [inline src/slides/class3/slides/ArrayMethods.jsx:84]: What do you mean by `función de prueba`?

File: src/slides/class3/slides/ArrayMethods.jsx [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+    </Slide>
+
+    <Slide backgroundColor="#0d0d0d" style={{ display: "flex" }}>
+      <Heading margin="0px" fontSize="50px">
+        Array.forEach()
+      </Heading>
+
+      <Text fontSize={30}>
+        El método <strong>forEach()</strong> ejecuta la función indicada una vez
+        por cada elemento del array..
```
Comment (JulianPasquale) [inline src/slides/class3/slides/ArrayMethods.jsx:111]:
```suggestion
        por cada elemento del array.
```

File: src/slides/class3/slides/ArrayMethods.jsx [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+      <Text fontSize={30}>La función reductora recibe cuatro argumentos:</Text>
+      <UnorderedList fontSize={30}>
+        <ListItem>Acumulador (acc)</ListItem>
+        <ListItem>Valor Actual (cur)</ListItem>
+        <ListItem>Índice Actual (idx)</ListItem>
+        <ListItem>Array (src)</ListItem>
+      </UnorderedList>
+      <Text fontSize={30}>
+        El valor devuelto de la función reductora se asigna al acumulador, cuyo
+        valor se recuerda en cada iteración de la matriz y, en última instancia,
```
Comment (JulianPasquale) [inline src/slides/class3/slides/ArrayMethods.jsx:164]:
```suggestion
        valor se recuerda en cada iteración y, en última instancia,
```

**Review by JulianPasquale — APPROVED** [review]

File: src/slides/class3/slides/ArrayMethods.jsx (hunk 1 above)
Comment (nachosoloaga) [reply src/slides/class3/slides/ArrayMethods.jsx:57]: Yes! I'll add them

File: src/slides/class3/slides/ArrayMethods.jsx (hunk 2 above)
Comment (nachosoloaga) [reply src/slides/class3/slides/ArrayMethods.jsx:84]: It's something like the function that elements will have to "pass" or "test". Maybe I'll rephrase that to `... que evalúa a true la función proporcionada`

**Pull request merged**